code-challenges/301/regex1/challenges-04.test.js

```diff
@@ -0,0 +1,199 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named isNum that takes in a string or number of any length. This function should use a regular expression pattern to return true if the input contains a number, and false if the input does not contain a number.
+
+For example: 
+12345 returns true
+'12345' returns true
+'h3llo world' returns true
+'hello world' returns false
+------------------------------------------------------------------------------------------------ */
+
+const isNum = (input) => {

```

Comment:
For this challenge, you will want to use the regex test method, since it will return a boolean value

```const isNum = (input) => {
  return /\d/.test(input);
};```